code/game/objects/items/weapons/material/misc.dm

```diff
@@ -70,6 +70,48 @@
 	icon_state = "unathiknife"
 	attack_verb = list("ripped", "torn", "cut")
 
+/obj/item/material/hatchet/machete
+	name = "machete"
+	desc = "A long, sturdy blade with a rugged handle. Leading the way to cursed treasures since before space travel."
+	icon = 'icons/obj/item/melee/machete.dmi'
+	item_icons = list(
+		slot_l_hand_str = 'icons/obj/item/melee/machete.dmi',
+		slot_r_hand_str = 'icons/obj/item/melee/machete.dmi',
+	)
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	contained_sprite = TRUE
+	icon_state = "machete"
+	item_state = "machete"
+	worn_overlay = "handle"
+	w_class = ITEMSIZE_NORMAL
+	slot_flags = SLOT_BELT|SLOT_HOLSTER
+	default_material = MATERIAL_TITANIUM
+	max_force = 20
+	force_divisor = 0.2
+	build_from_parts = TRUE
+
+/obj/item/material/hatchet/machete/unbreakable
+	unbreakable = TRUE
+
+/obj/item/material/hatchet/machete/steel
+	name = "fabricated machete"
+	desc = "A long, machine-stamped blade with a somewhat ungainly handle. Found in military surplus stores, malls, and horror movies since before interstellar travel."
+	default_material = MATERIAL_STEEL
+	matter = list(MATERIAL_STEEL = 15000, MATERIAL_PLASTIC = 2500)
+
+/obj/item/material/hatchet/machete/Initialize()
+	. = ..()
+	if(build_from_parts) 
+		var/image/I = image(icon, icon_state = "machete_handle")
+		I.color = pick(COLOR_BLUE, COLOR_RED, COLOR_CYAN, COLOR_BLACK, COLOR_AMBER, COLOR_GREEN, COLOR_OLIVE, COLOR_GUNMETAL, COLOR_DARK_BLUE_GRAY)
+		add_overlay(I)
```

Comment:
```suggestion
	build_from_parts = TRUE

/obj/item/material/hatchet/machete/Initialize()
	. = ..()
	if(build_from_parts) 
		var/image/I = image(icon, icon_state = "machete_handle")
		I.color = pick(COLOR_BLUE, COLOR_RED, COLOR_CYAN, COLOR_BLACK, COLOR_AMBER, COLOR_GREEN, COLOR_OLIVE, COLOR_GUNMETAL, COLOR_DARK_BLUE_GRAY)
		add_overlay(I)

/obj/item/material/hatchet/machete/unbreakable
	unbreakable = TRUE

/obj/item/material/hatchet/machete/steel
	name = "fabricated machete"
	desc = "A long, machine-stamped blade with a somewhat ungainly handle. Found in military surplus stores, malls, and horror movies since before interstellar travel."
	default_material = MATERIAL_STEEL
	matter = list(MATERIAL_STEEL = 15000, MATERIAL_PLASTIC = 2500)
```